src/main/java/com/iesmb/gestionalumnos/Entity/Curso.java

```diff
@@ -9,14 +9,18 @@ public class Curso {
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String materia;
-    private String profesor;
+    private String id_profesor;
```

Comment:
Se debe retirar este atributo que ya ha quedado obsoleto, como también su getter y setter

Reply:
Muchas gracias Profe, se me ha pasado eliminar el atributo, aplique las correciones y subi el commit "Segunda Issue - Relaciones CORREGIDA"

Reply:
De igual manera recuerde que esta issue estaba "repetida" con otra provista a mi compañero Laureano Chirino, ambas issues tenían solicitadas relaciones diferentes, la de el una one to one y la mía una one to many, posiblemente subir ambas causarían conflictos/anomalías